src/systems/hubs-systems.js

```diff
@@ -0,0 +1,30 @@
+import { CursorTargettingSystem } from "./cursor-targetting-system";
+import { ConstraintsSystem } from "./constraints-system";
+import { TwoPointStretchingSystem } from "./two-point-stretching-system";
+import { SingleActionButtonSystem, HoldableButtonSystem, HoverButtonSystem } from "./button-systems";
+import { HoverMenuSystem } from "./hover-menu-system";
```

Comment:
There's no point in calling all of these "systems" and then calling their "tick" function. They might as well just be functions who are passed the data they need in order to have the side effects they want to have. (Usually, that data is the interaction system state.)